src/domain/usecases/CopyDataSetsOUUserCase.ts

```diff
@@ -0,0 +1,96 @@
+import _ from "lodash";
+import { DataSetsRepository } from "domain/repositories/DataSetsRepository";
+import { PostOptions, Ref } from "types/d2-api";
+import { DataSet } from "domain/entities/DataSet";
+import { OUCopyResult } from "domain/entities/OUCopyResult";
+
+export class CopyDataSetsOUUserCase {
+    constructor(
+        private dataSetsRepository: DataSetsRepository
+    ) { }
+
+    async execute(options: {
+        originDataset: string;
+        destinationDatasets: string[];
+        replace: boolean | undefined;
+    }): Promise<OUCopyResult> {
+        const { originDataset, destinationDatasets, replace } = options;
+
+        console.debug(`Replace the destination OU: ${replace ? "True" : "False"}`);
```

Comment:
We can use `log.debug` (from utils/log)

Reply:
That's what I tried at first, but it didn't showed up in the terminal. Turns out that log doesn't work (except log.info) without a .env file. That's how its defined in utils/log.ts so i guess its intended this way.

Switching console.* to log.*

Reply:
Yeah, `info` is the default log level.

https://github.com/EyeSeeTea/d2-tools/blob/feature/datasets_copy-org-units/src/utils/log.ts#L12

It can be changed using an environment variable: `LOG_LEVEL=debug yarn start ...`

I'll add a note in the README